libs/core/kiln_ai/datamodel/test_basemodel.py

```
@@ -1,15 +1,19 @@
 import datetime
 import json
+import os
```

Ruff (0.11.9)

`os` imported but unused

Remove unused import: `os`

(F401)